cms/envs/common.py

```diff
@@ -89,7 +89,9 @@
     FILE_UPLOAD_STORAGE_BUCKET_NAME,
     FILE_UPLOAD_STORAGE_PREFIX,
 
-    COURSE_ENROLLMENT_MODES
+    COURSE_ENROLLMENT_MODES,
+
+    SITE_SUPPORT_LINK
```

Comment:
It needs to be in this PR (and #15017, actually) for the cms tests to pass -- the change will vanish after the first of the two branches merges to master and the second is rebased.